src/main/java/com/mycodefu/vsssfshss/Entrypoint.java

```diff
@@ -30,6 +37,32 @@ public void serverConnectionMessage(ChannelId id, MessageSender messageSender,
                     public void serverConnectionClosed(ChannelId id, MessageSender messageSender) {
 
                     }
+
+                    @Override
+                    public HttpMessage serverHttpMessage(ChannelId id, String ip, String path,
+                            Map<String, List<String>> queryParameters,
+                            Map<String, String> requestHeaders) {
+                        byte[] content = HttpResourceManager
+                                .getResource(path.equals("/") ? "/index.html" : path);
+                        if (content != null) {
+                            Map<String, Object> headers = new HashMap<>();
+                            headers.put("content-length", content.length);
+                            headers.put("content-type", "text/html");
+                            return new HttpMessage(HttpResponseStatus.OK, content, headers);
+                        } else {
+                            content = """
```

Comment:
Why don't we create a file in the resources called '404.html' and serve that rather than a string literal content response?

Reply:
done